src/app/developer/routes.ts

```diff
@@ -1,11 +1,16 @@
 import pathToRegexp from 'path-to-regexp';
+import Loadable from 'react-loadable';
 import { Route } from '../navigation/types';
-import Home from './components/Home';
 
 const ROUTES_PATH = '/';
 
 export const routes: Route<undefined, undefined> = {
-  component: Home,
+  component: Loadable({
+    loader: () => import(/* webpackChunkName: "DeveloperHome" */ './components/Home'),
+    loading: () => null,
```

Comment:
Similarly, can in indicator of loading/loaded be included?

And since this is the error page, maybe an `alert` (or some other fallback) if it fails to load?

Reply:
this is developer home so this route we can probably ignore